src/components/ebay-infotip/test/test.browser.js

```diff
@@ -0,0 +1,47 @@
+const sinon = require('sinon');
+const expect = require('chai').expect;
+const testUtils = require('../../../common/test-utils/browser');
+const renderer = require('../');
+
+describe('given the default infotip', () => {
```

Comment:
We should probably test for that the tooltip is visible instead of just testing for the emitted events.

Reply:
Yeah, I was having some trouble with this earlier. I'll look at it in the morning.